acceptance/tests/utf8/utf8-in-function-args.rb

```diff
@@ -4,93 +4,93 @@
-    agents.each do |agent|
-      step 'alert' do
-        result = on(
-          agent,
-          puppet("apply", "-e" "'alert(\"alert #{utf8chars}\")'"),
-          :environment => {:LANG => "en_US.UTF-8"}
-        )
-        assert_match(
-          /#{utf8chars}/,
-          result.stderr,
-          "Did not find the utf8 chars."
-        )
-      end
-    
-      step 'assert_type' do
-        on(
-          agent,
-          puppet(
-            "apply", "-e", "'notice(assert_type(String, \"#{utf8chars}\"))'"
-          ),
-          {
-            :environment => {:LANG => "en_US.UTF-8"},
-            :acceptable_exit_codes => [0],
-          }
-        )
-        on(
-          agent,
-          puppet("apply", "-e 'notice(assert_type(Float, \"#{utf8chars}\"))'"),
-          {
-            :environment => {:LANG => "en_US.UTF-8"},
-            :acceptable_exit_codes => [1],
-          }
-        )
-      end
-    
-      step 'filter' do
-        puppet_cmd = "'
-          [$a] = [[\"abc\", \"#{utf8chars}\", \"100\"]];
-          [$f] = [filter($a) |$p| {$p =~ /#{utf8chars}/}];
-          notice(\"f = $f\")
-        '"
-        result = on(
-          agent,
-          puppet("apply", "-e", puppet_cmd),
-          :environment => {:LANG => "en_US.UTF-8"}
-        )
-        assert_match(/#{utf8chars}/, result.stdout, "filter() failed.")
-      end
-    
-      agent_lookup_test_dir =
-        (agent == master) ?
-        master_lookup_test_dir :
-        agent_lookup_test_dir = agent.tmpdir("lookup_test_dir")
+  agents.each do |agent|
+    step 'alert' do
+      result = on(
+        agent,
+        puppet("apply", "-e" "'alert(\"alert #{utf8chars}\")'"),
+        :environment => {:LANG => "en_US.UTF-8"}
+      )
+      assert_match(
+        /#{utf8chars}/,
+        result.stderr,
+        "Did not find the utf8 chars."
+      )
+    end
+  
+    step 'assert_type' do
+      on(
+        agent,
+        puppet(
+          "apply", "-e", "'notice(assert_type(String, \"#{utf8chars}\"))'"
+        ),
+        {
+          :environment => {:LANG => "en_US.UTF-8"},
+          :acceptable_exit_codes => [0],
+        }
+      )
+      on(
+        agent,
+        puppet("apply", "-e 'notice(assert_type(Float, \"#{utf8chars}\"))'"),
+        {
+          :environment => {:LANG => "en_US.UTF-8"},
+          :acceptable_exit_codes => [1],
+        }
+      )
+    end
+  
+    step 'filter' do
+      puppet_cmd = "'
+        [$a] = [[\"abc\", \"#{utf8chars}\", \"100\"]];
+        [$f] = [filter($a) |$p| {$p =~ /#{utf8chars}/}];
+        notice(\"f = $f\")
+      '"
+      result = on(
+        agent,
+        puppet("apply", "-e", puppet_cmd),
+        :environment => {:LANG => "en_US.UTF-8"}
+      )
+      assert_match(/#{utf8chars}/, result.stdout, "filter() failed.")
+    end
+  
+    agent_lookup_test_dir =
+      (agent == master) ?
+      master_lookup_test_dir :
+      agent_lookup_test_dir = agent.tmpdir("lookup_test_dir")
```

Comment:
This can be simplified to just `agent_lookup_test_dir = agent.tmpdir("lookup_test_dir")`